_posts/en/newsletters/2024-12-06-newsletter.md

```diff
@@ -0,0 +1,117 @@
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #30708][] adds the `getdescriptoractivity` RPC command that
+  retrieves all transactions associated with a [descriptor][topic descriptors]
+  within a specified set of blockhashes, allowing wallets to interact with
+  Bitcoin Core in a stateless manner. This command is particularly useful when
+  used in conjunction with `scanblocks` (See Newsletter [#222][news222
+  scanblocks]), which identifies the set of blockhashes containing transactions
+  associated with a descriptor.
+
+- [Core Lightning #7832][] introduces [anchor outputs][topic anchor outputs] for
+  non-urgent unilateral close transactions, starting with a block target of 2016
+  blocks (approximately 2 weeks) and gradually reducing to 12 blocks. The
+  broadcast timestamp will be tracked to ensure consistent behaviour across
+  restarts. Previously, these transactions did not have anchor outputs by
+  default, making it difficult to add one manually, or impossible to use
+  [CPFP][topic cpfp] fee bumping if left without an anchor.
+
+- [LND #8270][] implements the [channel quiescence][topic channel commitment
+  upgrades] protocol as specified in [BOLT2][] (see Newsletter [#309][news309
+  quiescence]), which is a prerequisite for [dynamic commitment][topic channel
+  commitment upgrades] and [splicing][topic splicing]. The protocol allows a
+  node to both respond to a peer's quiescence request and initiate the process
+  using new `ChannelUpdateHandler` operations. This PR also adds a configurable
+  timeout mechanism to handle unresponsive peers by disconnecting them if the
+  quiescent state persists for too long without resolution.
+
+- [LND #8390][] introduces support for setting and relaying an experimental
+  [HTLC endorsement][topic htlc endorsement] signaling field in
+  `update_add_htlc` messages, aimed at researching [channel jamming
+  attack][topic channel jamming attacks] prevention. If a node receives an HTLC
+  with the signaling field, it will relay the field as-is; otherwise, it sets a
+  default value of zero. This feature is enabled by default but can be disabled.
+
+- [BIPs #1534][] merges BIP349 for the specification of `OP_INTERNALKEY`, a
+  new BIP342 tapscript only opcode that puts the taproot's internal key on the
+  stack. Authors of scripts need to know the internal key before they can pay to
+  an output, so they could include an internal key directly in a script. See
```

Comment:
Just to confirm: is this sentence right? I am a bit lost in it but that may be due to my lack of knowledge.

Reply:
It's correct but I'll add some clarification.